tensorflow/python/tools/print_selective_registration_header.py

```diff
@@ -14,13 +14,21 @@
 # ==============================================================================
 r"""Prints a header file to be used with SELECTIVE_REGISTRATION.
 
-Example usage:
-  print_selective_registration_header \
-      --graphs=path/to/graph.pb > ops_to_register.h
+An example of command-line usage is:
+  bazel build tensorflow/python/tools:print_selective_registration_header && \
+  bazel-bin/tensorflow/python/tools:print_selective_registration_header \
+    --graphs=path/to/graph.pb > ops_to_register.h
 
-  Then when compiling tensorflow, include ops_to_register.h in the include
-  search path and pass -DSELECTIVE_REGISTRATION  - see
-  core/framework/selective_registration.h for more details.
+Then when compiling tensorflow, include ops_to_register.h in the include
+search path and pass -DSELECTIVE_REGISTRATION  - see
```

Comment:
Of course (my fault), which brings up the question (again), why there even is this disconnect? Since the original docs only mention `-DSELECTIVE_REGISTRATION`, it seems that `-DSUPPORT_SELECTIVE_REGISTRATION` came up later in the process.